gitlab/pkg/reconciler/gitlabsource.go

```diff
@@ -233,7 +149,7 @@ func (r *Reconciler) reconcile(source *sourcesv1alpha1.GitLabSource) error {
 		}
 	}
 
-	ksvc, err = r.waitForKnativeServiceReady(source)
+	ksvc, err = r.waitForKnativeServiceReady(source) // TODO: remove this. setup a watch on the service instead.
```

Comment:
Can we create issues to track this and the other TODO?

Reply:
https://github.com/knative/eventing-contrib/issues/1066